CefSharp/Internals/JavascriptObjectRepository.cs

```diff
@@ -119,6 +119,13 @@ public bool TryGetProperty(long objectId, string name, out object result, out st
                 throw new InvalidOperationException(string.Format("Property {0} not found on Object of Type {1}", name, obj.Value.GetType()));
             }
 
+            if (property.JsObject != null)
+            {
+                obj = property.JsObject.Bind();
+                result = !obj.IsNull && obj.Value.GetType().IsArray ? obj.Value : obj;
```

Comment:
OK to add IsArray, but no need to serialize it. In fact, if is array, the object is not passed to browser, only value which is the array is.